src/connectionController.ts

```diff
@@ -458,15 +459,25 @@ export default class ConnectionController {
 
   // Used to re-authenticate with OIDC.
   async _reauthenticationHandler() {
+    console.log('----------------------');
```

Comment:
Some leftover logs?

Reply:
Logs in use 😄 but these I cleared up for now, don't be surprised if you see more. Added WIP label to the pr.